wolox-books/src/app/screens/register/register.component.html

```diff
@@ -0,0 +1,32 @@
+<div class="register">
+  <img class="register__logo" src="/assets/images/logo-wolox.png" alt="Logo Wolox">
+  <form [formGroup]="registerForm" (ngSubmit)="registerUser(registerForm.value)" class="register__form">
+    <div class="register__name form-input">
+      <label class="form-input__label" for="name">Nombre</label>
+      <input class="form-input__input" name="name" id="name" formControlName="name">
+    </div>
+    <div class="register__last-name form-input">
+      <label class="form-input__label" for="last-name">Apellido</label>
+      <input class="form-input__input" name="last-name" id="last-name" formControlName="last-name">
+    </div>
+    <div class="register__email form-input">
+      <label class="form-input__label" for="email">Email</label>
+      <input class="form-input__input" type="email" name="email" id="email" formControlName="email">
+      <span class="form-input__error" *ngIf="registerForm.controls['email']?.errors?.required && registerForm.controls['email']?.touched">El email es requerido</span>
+      <span class="form-input__error" *ngIf="registerForm.controls['email']?.errors?.email && registerForm.controls['email']?.touched && registerForm.controls['email']?.value?.length > 5">Formato de email incorrecto</span>
+    </div>
+    <div class="register__password form-input">
+      <label class="form-input__label" for="password">Password</label>
+      <input class="form-input__input" type="password" name="password" id="password" formControlName="password">
+      <span class="form-input__error" *ngIf="registerForm.controls['password']?.errors?.required && registerForm.controls['password']?.touched">El password es requerido</span>
+    </div>
+    <div class="register__password-confirmation form-input">
+      <label class="form-input__label" for="password-confirmation">Confirmación de Password</label>
+      <input class="form-input__input" type="password" name="password-confirmation" id="password-confirmation" formControlName="password-confirmation">
```

Comment:
when you have long lines it is a good idea to do this to improve readability

```suggestion
      <input
        class="form-input__input"
        type="password"
        name="password-confirmation"
        id="password-confirmation"
        formControlName="password-confirmation">
```

Reply:
I tried...